### DISCUSSION TITLE
SyntaxError: Unexpected token { in JSON at position 73

### DISCUSSION THREAD

**Discussion by Bunker56** [PhlexPlexico/G5V#140] [Q&A] [answered]
I'm sorry, maybe this question has already been asked, but I just didn't find it. When creating a match, I got the following error - "SyntaxError: Unexpected token {in JSON at position 73". I've already met her, but it was a long time ago and I just don't remember how I fixed it. I would be very grateful for a hint.

G5API          | 0|G5API  | Error on isAvailable server: SyntaxError: Unexpected token { in JSON at position 73
G5API          | 0|G5API  | SyntaxError: Unexpected token { in JSON at position 73
G5API          | 0|G5API  |     at JSON.parse (<anonymous>)
G5API          | 0|G5API  |     at ServerRcon.isGet5Available (file:///Get5API/G5API/utility/serverrcon.js:61:39)
G5API          | 0|G5API  |     at processTicksAndRejections (node:internal/process/task_queues:96:5)
G5API          | 0|G5API  |     at async file:///Get5API/G5API/routes/matches/matches.js:842:10
G5API          | 0|G5API  | POST /matches/ 500 38.803 ms - 68

**Comment by PhlexPlexico** [comment]
Check your server logs and do a status check on the server. Make sure you have the SteamWorks extension installed, as well as Rest In Pawn. Those are the two extensions required by get5 and the API. I believe that fixes 90% of problems with syntax errors.

Also, this is more of an API issue and you may find answers in the closed issues in that repo, too :)

**Comment by Bunker56** [comment]
It looks like everything was solved by removing the get5_apistats plugin...

**Reply by PhlexPlexico** [reply]
You mean the old plugin? If you wish to use the, you have to use G5WS as well, or else you won't be able to interact with the game server.

**Comment by Bunker56** [comment, marked as answer]
It was another admin who tried, and I broke my whole head, what could have happened... Thank you!!! :)
